makefiles/dbus.mk

```
@@ -0,0 +1,117 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS  += dbus
+DBUS_VERSION := 1.12.20
+DEB_DBUS_V   ?= $(DBUS_VERSION)
+
+ifeq (,$(findstring darwin,$(MEMO_TARGET)))
+DBUS_PROFILE := /etc/profile.d
```

iphoneos-arm64-rootless
```suggestion
DBUS_PROFILE := $(MEMO_PREFIX)/etc/profile.d
```